LogTarget.php

```diff
@@ -165,6 +165,21 @@ protected function collectSummary()
     }
 
     /**
+     * Returns request time in seconds
+     *
+     * @return float request time in seconds.
+     * @see http://php.net/manual/en/function.microtime.php#example-2550
+     * @since 2.0.8
+     */
+    protected function getRequestTime()
```

Comment:
Why do we need this as separated method at all then?
Why do not simply put it inside `collectSummary()`:

```php
'time' => isset($_SERVER['REQUEST_TIME_FLOAT']) ? $_SERVER['REQUEST_TIME_FLOAT'] : microtime(true),
```